scan/src/components/proposal/TurnoutChart.re

```diff
@@ -0,0 +1,60 @@
+module Styles = {
+  open Css;
+
+  let chartContainer =
+    style([
+      position(`relative),
+      width(`px(208)),
+      height(`px(208)),
+      Media.mobile([width(`px(140)), height(`px(140))]),
+    ]);
+
+  let textContainer =
+    style([
+      position(`absolute),
+      backgroundColor(Colors.white),
+      top(`px(8)),
+      left(`px(8)),
+      right(`px(8)),
+      bottom(`px(8)),
+      borderRadius(`percent(50.)),
+    ]);
+  let circle = percent => {
+    style([
+      width(`percent(100.)),
+      height(`percent(100.)),
+      backgroundColor(Colors.profileBG),
+      borderRadius(`percent(50.)),
+      selector(
+        "> circle",
+        [
+          SVG.fill(Colors.profileBG),
+          SVG.strokeWidth(`px(16)),
+          SVG.stroke(Colors.bandBlue),
+          //TODO: it will be remove when the bs-css upgrade to have this proporty
+          unsafe("stroke-dasharray", "calc(" ++ percent ++ "* 653.45 / 100) 653.45"),
```

Comment:
Let's use `{j|calc($percent*653.45 / 100) 653.45|j}`.
And show how to get this value (I mean 653.45).